Makefile

```diff
@@ -28,10 +28,10 @@ test_lint:
 # for more than once in Travis. Change `make test` in .travis.yml
 # to `make test_multi`.
 test_multi:
-	cd network; \
+	cd services/skipchain; \
```

Comment:
? Why testing skipchain only here ? either test everything (or only services) or change the name to `test_skipchain` if that's really needed (I think not).